src/Microsoft.AspNet.Mvc.DataAnnotations/DataAnnotationsModelValidatorProvider.cs

```diff
@@ -19,18 +20,23 @@ public class DataAnnotationsModelValidatorProvider : IModelValidatorProvider
     {
         private readonly IOptions<MvcDataAnnotationsLocalizationOptions> _options;
         private readonly IStringLocalizerFactory _stringLocalizerFactory;
+        private readonly IValidationAttributeAdapterProvider _validationAttributeAdapterProvider;
 
         /// <summary>
         /// Create a new instance of <see cref="DataAnnotationsModelValidatorProvider"/>.
         /// </summary>
         /// <param name="options">The <see cref="IOptions{MvcDataAnnotationsLocalizationOptions}"/>.</param>
         /// <param name="stringLocalizerFactory">The <see cref="IStringLocalizerFactory"/>.</param>
+        /// <remarks><paramref name="options"/> and <paramref name="stringLocalizerFactory"/>
```

Comment:
In regular use, DI creates instances and extra ctor overloads confuses our CI system (unless that's been improved?).